fronts-client/src/components/feed/RecipeSearchContainer.tsx

```diff
@@ -31,24 +32,56 @@ const FixedContentContainer = styled.div`
 
 interface Props {
   rightHandContainer?: React.ReactElement<any>;
-  recipes: Record<string, Recipe>;
-  chefs: Record<string, Chef>;
 }
 
 enum FeedType {
   recipes = 'recipeFeed',
   chefs = 'chefFeed',
 }
 
-const FeastSearchContainerComponent = ({
-  rightHandContainer,
-  recipes,
-  chefs,
-}: Props) => {
+export const RecipeSearchContainer = ({ rightHandContainer }: Props) => {
   const [selectedOption, setSelectedOption] = useState(FeedType.recipes);
-  const handleOptionChange = (optionName: FeedType) => {
-    setSelectedOption(optionName);
-  };
+  const [searchText, setSearchText] = useState('');
+  const recipes: Record<string, Recipe> = useSelector((state: State) =>
+    recipeSelectors.selectAll(state)
+  );
+  const dispatch: Dispatch = useDispatch();
+  const fetchChefs = useCallback(
+    (params: object, isResource: boolean) => {
+      dispatch(fetchLive(params, isResource));
+    },
+    [dispatch]
+  );
+  const chefSearchIds = useSelector((state: State) =>
+    chefSelectors.selectLastFetchOrder(state)
+  );
+
+  const debouncedRunSearch = debounce(() => runSearch(), 750);
+
+  useEffect(() => {
+    debouncedRunSearch();
+  }, [selectedOption, searchText]);
+
+  const getParams = (query: string) => ({
+    'web-title': query,
+    'page-size': '20',
+    'show-elements': 'image',
+    'show-fields': 'all',
+  });
+
+  const runSearch = useCallback(
+    (page = 1) => {
```

Comment:
Am I right to think this default value for page is always used? Presumably this means you can only see the first page of results for now.